src/components/inline/InputText/InputText.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import {CSSProperties, useState} from "react";
```

Comment:
We should make sure that Prettier/ESLint is set up right for TypeScript files. They normally prefer to put a space on the insides of `{` and `}`.